test/unit/spec/EnrollChoices_spec.js

```diff
@@ -327,7 +336,7 @@ function (_, $, OktaAuth, Util, EnrollChoicesForm, Beacon, Expect, Router,
         itp('has a setup button for each unenrolled optional factor which navigates to the correct page (On-Prem)',
           function () {
           return setupWithAllOptionalSomeEnrolled(true).then(function (test) {
-            expect(test.form.factorButton('OKTA_VERIFY').length).toBe(1);
+            expect(test.form.factorButton('OKTA_VERIFY_PUSH').length).toBe(0);
```

Comment:
Ah makes sense. Would it be a good idea then to add QUESTION and check that the Setup button is there for it?